src/selectors/lesson.js

```diff
@@ -38,3 +38,32 @@ export const getFilteredAndIndexedLessons = createSelector(
     }, {});
   }
 );
+
+export const getAvailableLessons = createSelector(
+  [getFilter, getFilteredLessons],
+  (current_filter = {}, filteredLessons = {}) => {
+
+    let availableLessons = {};
+    for (let tag in current_filter.operativsystem) {
+      availableLessons[tag] = 0;
+    }
+    for (let tag in current_filter.tema) {
+      availableLessons[tag] = 0;
```

Comment:
Er vi sikre på at det er noe vi ønsker å tillate, da? For meg høres det nesten ut som det bør fikses i andre enden. Altså at hvis du prøver å opprette et tema med samme navn som et OS så bør du legge det inn som OS i stedet.